src/lib.rs

```diff
@@ -1017,6 +1017,22 @@ impl Element {
         })
     }
 
+    /// Simulate the user sending keys to an element.
+    ///
+    /// Note that since this *may* result in navigation, we give up the handle to the element.
```

Comment:
I think this should probably just take `&mut self` instead, and return `impl Future<Item = ()`, since sending keys is unlikely to cause navigation directly.

Reply:
Sure, we can do that